Ginger/GingerCoreCommon/Repository/RepositoryItemBase.cs

```diff
@@ -188,7 +188,7 @@ public static string GetClassShortName(Type t)
         /// <returns></returns>
         public virtual string GetItemType()
         {
-            throw new Exception("Unknown Type for GetItemType " + this.GetType().Name);
+            return "Unknown Type for GetItemType " + this.GetType().Name;
```

Comment:
As we are changing Method implementation in base class, it may have some other consequences, Keep the throw exception and catch this exception where it needs to be handled

Reply:
okay